Services/BlockedTokenManager.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Lexik\Bundle\JWTAuthenticationBundle\Services;
+
+use DateInterval;
+use DateTimeImmutable;
+use DateTimeZone;
+use Lexik\Bundle\JWTAuthenticationBundle\Exception\MissingClaimException;
+use Psr\Cache\CacheItemPoolInterface;
+
+class BlockedTokenManager
```

Comment:
Like I suggested in the discussion, this would probably be good to add an interface for, that way folks can change the storage backend entirely and not rely on a cache store if they want some other method for storing/tracking these.